systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaUtils.java

```diff
@@ -34,11 +34,13 @@ public static void waitUntilKafkaCRIsReady(String clusterName) {
         LOGGER.info("Kafka CR will be ready");
     }
 
-    public static void waitUntilKafkaStatusConditionIsPresent(String clusterName) {
+    public static void waitUntilKafkaStatusConditionIsNotReady(String clusterName) throws InterruptedException {
         LOGGER.info("Waiting till kafka resource status is present");
         TestUtils.waitFor("Waiting for Kafka resource status is ready", Constants.GLOBAL_POLL_INTERVAL, Constants.GLOBAL_TIMEOUT,
-            () ->  Crds.kafkaOperation(kubeClient().getClient()).inNamespace(kubeClient().getNamespace()).withName(clusterName).get().getStatus().getConditions().get(0) != null
+            () ->  Crds.kafkaOperation(kubeClient().getClient()).inNamespace(kubeClient().getNamespace()).withName(clusterName).get().getStatus().getConditions().get(0).getType().equals("NotReady") &&
```

Comment:
Maybe I'm reading it wrong. But we have:
* Method named `waitUntilKafkaStatusConditionIsNotReady`
* a Log message _Waiting till kafka resource status is present_
* some other message saying _Waiting for Kafka resource status is ready_
* A condition which seems to check if the pod has condition `NotReady`

So I wonder which of these does it actually do.

Reply:
Yeah i see, i forgot rewrite logs. This usage of method is basically, when we expect that something will did not work and from status Ready will change to NotReady. Logs rewrited and sorry for creating confusion.